src/__main__.py

```diff
@@ -1,3 +0,0 @@
-import runpy
```

Comment:
So I take it that this file is no longer needed? I trust you there - did not test locally.

Reply:
It uses `-m "particle.__main__:main"` in the zip app command instead (which writes this file for you) and that way you can't accidentally put this into the PyPI package (which was happening after I moved the files because I forgot about it!)

Reply:
OK. Subtle. Thanks.